### PR TITLE
[torch] add erfc & erfinv

### PR DESCRIPTION
**Pull request by khushi-411**

### CODE REVIEW

File: ivy/functional/frontends/torch/pointwise_ops.py [hunk 1]
```diff
@@ -520,6 +520,18 @@ def erf(input, *, out=None):
...
+@with_unsupported_dtypes({"2.0.1 and below": ("float16", "complex")}, "torch")
+@to_ivy_arrays_and_back
+def erfc(input, *, out=None):
+    return 1 - ivy.erf(input, out=out)
+
+
+@with_unsupported_dtypes({"2.0.1 and below": ("float16", "complex")}, "torch")
+@to_ivy_arrays_and_back
+def erfinv(input, *, out=None):
+    return ivy.erfinv(input, out=out)
```
Comment (khushi-411) [inline ivy/functional/frontends/torch/pointwise_ops.py:532]: TODO (Note to self): use composite funtion

**Comment by khushi-411** [comment]
Ping: @RickSanchezStoic :)

**Comment by RickSanchezStoic** [comment]
Hey @khushi-411  something wrong with the tests, I think it's everywhere, I am currently looking into this and others are too hopefully, so will get back to this.

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**